internal/security/proxy/service.go

```diff
@@ -96,23 +95,12 @@ func (s *Service) Init(cert CertificateLoader) error {
 		return err
 	}
 
-	services := config.ListDefaultServices()
-	//TODO: There are specific other services being routed through Kong. This should probably be driven through configuration
-	//      For now, I'm putting this inline
-	services["edgex-support-rulesengine"] = "Rulesengine"
-	services["edgex-device-virtual"] = "Virtualdevice"
-
-	for _, name := range services {
-		params := Configuration.Clients[name]
-		if params.Host == "" {
-			LoggingClient.Warn("missing config information for service: " + name)
-			continue
-		}
+	for clientName, client := range Configuration.Clients {
```

Comment:
@anonymouse64 -- are you OK with this implementation?

It walks the clients defined in configuration and establishes a route for each of them in Kong. Prior implementation used a standard list of services -- a warning was logged if one of the standard services was not defined in configuration but execution was allowed to proceed.

Updated implementation drops the standard service list and corresponding warning. Seems more flexible to me in that we can define -- via configuration -- additional services outside the standard set (if that was desirable).